test/module.js

```diff
@@ -0,0 +1,134 @@
+var tests = {};
```

Comment:
We already have a `test/main/module.js`, so I'd prefer this be name `module-only.js` or something like that to avoid them being confused.